bears/vcs/git/GitCommitBear.py

```diff
@@ -96,7 +96,9 @@ def get_host_from_remotes():
             netloc = urlparse(url)[1]
         return netloc.split('.')[0]
 
-    def run(self, allow_empty_commit_message: bool = False, **kwargs):
+    def run(self, allow_empty_commit_message: bool = False,
+            check_revert_commits: bool = True,
```

Comment:
maybe to fit the naming scheme: `allow_revert_commits`? And it's by default `False`? (tbh I'm not sure what should be default for this anyway...)

Reply:
Ideally it should be true because this not only checks for revert commits but also that they have a reason too.